Packs/ApiModules/Scripts/CSVFeedApiModule/CSVFeedApiModule.py

```diff
@@ -181,21 +181,25 @@ def get_feed_content_divided_to_lines(self, url, raw_response):
         return response_content.decode(self.encoding).split('\n')
 
 
-def determine_indicator_type(indicator_type, default_indicator_type, value):
+def determine_indicator_type(indicator_type, default_indicator_type, auto_detect, value):
```

Comment:
Remind me why here we need both `indicator_type` and `default_indicator_type` - the JSONFeedApiModule only has the first. If I'm not mistaken, the `indicator_type` comes from the config which comes from whatever Feed integration that is using this API module - how they decide to determine the value there in their code is up to the integration, whether it be hardcoding it or allowing the user to set it via an integration parameter.

Reply:
Yes , In the jsonfeed api it is like you said, the indicator type comes from the config, but in csv it seems to be different and it might not come from config. That was the code before my changed , I did not change the login only added a check for auto_detect key.

Reply:
Please check. I think this is a redundancy.

Reply:
I checked. Can leave it as you did it but please update the `indicator_type` argument description in `FeedCSV.yml` to note that if `auto_detect_type` integration parameter has been marked true that the `indicator_type` argument of `csv-get-indicators` command will be ignored. The same goes for in the `FeedPlainText.yml`.

Reply:
I did it for both already.
- additionalinfo: Type of the indicator in the feed, If auto-detect is checked then the value set as Indicator Type will be ignored.

Reply:
Not the integration parameter tooltip - I know you updated the `additionalinfo` field there for both integrations. I asked that the description of the `indicator_type` argument of the `csv-get-indicators` and `plaintext-get-indicators` commands be updated.